### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [jeekayen/homeworksubmission#1]
The goal of this assignment is focus on

- Take a step further into visualization world.
- How to split your data into train and test part
- Correlation between each feature columns and label column
- Build your first model linear regression and try some methods to evaluate (MAE, MSE, RMSE)
- Make some insights and conclusion about the data

# Things you did well:

- You actually accomplished the whole assignment, you are awesome.
- You fluently make use of slicing operator ":" with iloc() ,very few people can actually remember using this at the beginning.
- Nice and clean code. It's really great help me to go through your work.
- Data science is not only about features, models, accuracy but also about actions. What we need to do after get the conclusion? You done this very well, make a good suggestion for the company!

Congrats on your hard work! Keep going and do great thing.

**Comment by coderschoolreview** [comment]
### The goal of this assignment is focus on
- Get to know about NLP techniques
- Understand the concept of classification problem
- Evaluate the classification result with Accuracy, Precision and Recall
- Build your first model logistic regression

### Things you did well:
- Understand the basic flow to handle classification task
- Be familiar with the techniques of NLP
- You get used to dealing with train/test split concept and evaluating results with accuracy reports

### Things to work on:
- You need to understand the meaning of each step in this exercise (not only copy and paste) 
- Revise the notebook to be more comprehensive and extract more information from the results
- You need to understand more about the concept of training/testing set, why it matters and how it is utilized

To sum up, you have done a great job as usual though you are very busy :). Keep up the spirit with your final project \m/.
